### ISSUE TITLE
Clicking on Node Highlights Output Pin/Variable Name

### ISSUE THREAD

**Issue by Nasuru** [brumenn/FlatNodes#12]
Hi, 

I made a question a couple days and you mentioned to put it here for the screenshots. And based on the source code, I was confused too on how its causing this behavior. Below is the default behavior where I clicked on the default node and just the yellow outline no highlighting pin. However, clicking on the flat node it automatically highlights the output pin or variable name and stays highlighted without needing to hover over and doesn't stop until clicked elsewhere.

Default:
![DefaultNode](https://github.com/brumenn/FlatNodes/assets/37818222/1d14e63c-b97d-41b3-a79e-04bd0b94b961)

FlatNodes:
![FlatNode](https://github.com/brumenn/FlatNodes/assets/37818222/f42e4cb9-6b17-4f44-a948-92b091820416)
![FlatNode2](https://github.com/brumenn/FlatNodes/assets/37818222/f898e056-f4ab-44cb-af63-21da2462f8cc)

**Comment by Nasuru** [comment]
Hi, have you gotten the chance to see this?

**Comment by brumenn** [comment]
Hi, yes absolutly, but I haven't managed to reproduce the issue on my side.. And I don't have enough time to dig for a problem that I can't identify.

So I keep the issue open for everyone.

What's your system config ?

**Comment by Nasuru** [comment]
Sorry, I didn't get the notification. I looked into it again after I saw your response because based on the code I didn't think it should've been happening, but it was only when Flat Nodes was enabled that it happened.

Well, it was actually Blueprint Assist plugin w Flat nodes enabled that causes it to highlight like that. I'll have to see what setting from it did that. Thanks for the response (and plugin :)) and have a good rest of your day!
